Assets/MRTK/SDK/Editor/Inspectors/UX/ObjectManipulator/ObjectManipulatorInspector.cs

```diff
@@ -103,13 +103,13 @@ public override void OnInspectorGUI()
             EditorGUILayout.PropertyField(allowFarManipulation);
 
             // Near Interaction Support foldout
-            nearInteractionFoldout = EditorGUILayout.Foldout(nearInteractionFoldout, "Near Interaction Support", false);
+            nearInteractionFoldout = EditorGUILayout.Foldout(nearInteractionFoldout, "Near Interaction Support", true);
 
             if (nearInteractionFoldout)
             {
                 if (instance.GetComponent<NearInteractionGrabbable>() == null)
                 {
-                    EditorGUILayout.HelpBox("By default, ObjectManipulator is a far interaction component.  Add a NearInteractionGrabbable component to enable near interaction support.", MessageType.Info);
+                    EditorGUILayout.HelpBox($"By default, {nameof(ObjectManipulator)} only responds to far interaction input.  Add a {nameof(NearInteractionGrabbable)} component to enable near interaction support.", MessageType.Warning);
```

Comment:
New Inspector appearance, changed message to warning:
![image](https://user-images.githubusercontent.com/53493796/105248087-ad595900-5b2a-11eb-83dc-b1c8fa34e0dc.png)